src/vdlm2.rs

```diff
@@ -290,6 +302,46 @@ pub enum AdscTags {
         ts_sec: f64,
         pos_accuracy_nm: f64,
         nav_redundancy: bool,
-        tcas_avail: bool
-    }
-}
\ No newline at end of file
+        tcas_avail: bool,
+    },
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, PartialOrd)]
+pub struct CPDLC {
+    pub err: bool,
+    pub atc_downlink_msg: Option<ATCDownlinkMsg>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, PartialOrd)]
+pub struct ATCDownlinkMsg {
+    pub header: ATCDownlinkMsgHeader,
+    pub atc_downlink_msg_element_id: ATCDownlinkMsgElementID,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, PartialOrd)]
+#[serde(rename_all = "snake_case")]
```

Comment:
Don't need this line here, as there's nothing here that needs to be renamed from what I can see.